src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  margin: 10px 0;
+}
```

Comment:
```suggestion
.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
}
```

Reply:
When I make all suppported changes in styles my tests doesn't work